compose/project.py

```diff
@@ -115,6 +117,14 @@ def from_dicts(cls, name, service_dicts, client, use_networking=False, network_d
                     net=net,
                     volumes_from=volumes_from,
                     **service_dict))
+        if config_data.volumes:
+            for vol_name, data in config_data.volumes.items():
+                project.volumes.append(
+                    Volume(
+                        client=client, project=name, name=vol_name,
+                        driver=data.get('driver'), driver_opts=data.get('driver_opts')
+                    )
```

Comment:
can we override this somehow? We prefix our volumes in volplugin with the tenant name, and I'm fairly certain others are doing similar syntaxes with the volume name.